### DISCUSSION TITLE
Devo realizar o Download do .NET 3.1 Core mesmo sem LTS?

### DISCUSSION THREAD

**Discussion by EuSouKaze** [balta-io/forum#260] [Tira Dúvidas] [answered]
Comecei agora os fundamentos do C# e estou no Modulo ".NET" na primeira aula "Instalação", segue o link abaixo.

https://balta.io/player/assistir/5ef614da-366d-6156-d049-694800000000/914e8f31-c6ec-442e-9cfa-000028010401

Eu compreendi que o curso é focado no .NET 3.1, mas também compreendi que nós sempre devemos optar por utilizar Frameworks que contenham LTS. Mas ao entrar na hora de realizar as instalações, descobri que o 3.1 deixou de ter suporte. E no próprio site da Microsoft contém a frase "_**As versões a seguir atingiram o fim da vida útil, o que significa que não há mais suporte para elas. É recomendável migrar para uma versão com suporte**_".

Minha duvida é: Eu devo mesmo assim fazer o download do .NET Core 3.1 já que o curso é focado nele e isso não ira me atrapalhar em nada durante o curso ou devo realizar o download de um Framework mais recente?

PS: Desculpe se a pergunta for boba ou se já foi respondida em outro momento (Não consegui encontrar).

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @EuSouKaze, como vai?

Pode instalar sim, sem problemas. Como comento no curso de Fundamentos do C#, as versões do .NET podem ser instaladas lado a lado, ou seja, você pode ter o 3.1, 5, 6 e 7 rodando na mesma máquina.
